mxcube3/core/components/user/usermanager.py

```diff
@@ -158,6 +154,7 @@ def login(self, login_id: str, password: str):
             if not "sid" in flask.session:
                 flask.session["sid"] = str(uuid.uuid4())
 
+            self.update_active_users()
```

Comment:
Moved `update_active_users` to before `login_user` so that potential still valid sessions are invalidated (if user is inactive)